appveyor.yml

```diff
@@ -21,24 +21,26 @@ environment:
       CONDA_PY: "27"
       ARCH: '64'
     - PYTHON: "C:\\Miniconda36"
-      CONDA_PY: "36"
+      CONDA_PY: "36" # No MDTraj for py37/Win32
       ARCH: '32'
-    - PYTHON: "C:\\Miniconda36-x64"
-      CONDA_PY: "36"
+    - PYTHON: "C:\\Miniconda37-x64"
+      CONDA_PY: "37"
       ARCH: '64'
 
 build: false
 
 install:
   # install python
   - SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%
-  # test python version
-  - "%CMD_IN_ENV% conda config --add channels omnia"  # TODO: switch to conda-forge?
-  - "%CMD_IN_ENV% conda update -yq --all"
-  - "%CMD_IN_ENV% conda install -yq \"conda-build<3.0.26\""
-  - "%CMD_IN_ENV% python --version"
-  - "%CMD_IN_ENV% python -c \"import struct; print(struct.calcsize('P') * 8)\""
-  # now we do our own install
+  - conda config --add channels conda-forge
+  - conda install -yq conda=4.5 conda-build=3.10
 
 test_script:
-  - "%CMD_IN_ENV% conda build --quiet ci\\conda-recipe"
+  #- "%CMD_IN_ENV% activate base"
+  #- "%CMD_IN_ENV% python --version"
+  #- "%CMD_IN_ENV% python -c \"import struct; print(struct.calcsize('P') * 8)\""
+  #- "%CMD_IN_ENV% conda build --quiet ci\\conda-recipe"
+  - activate base
+  - python --version
+  - python -c "import struct; print(struct.calcsize('P') * 8)"
```

Comment:
Is this leftover debug code or actually usefu/required?

Reply:
This verifies that we're on the right architecture (32-bit vs 64-bit), and so plays the same sort of role as the `python --version` above it. It dates back to some very old MDTraj version of the AppVeyor script. That said, I don't know how much longer we'll support 32-bit Windows. MDTraj stopped releasing for 32-bit a few releases ago, so I think we'll do the same soon.